pkg/azurestore/azureservice.go

```diff
@@ -309,3 +376,47 @@ func checkForNotFoundError(err error) error {
 	}
 	return err
 }
+
+// parse the Range, If-Match, If-None-Match, If-Unmodified-Since, If-Modified-Since headers if present
+func ParseDownloadOptions(r *http.Request) (*azblob.DownloadStreamOptions, error) {
+	input := azblob.DownloadStreamOptions{AccessConditions: &azblob.AccessConditions{}}
+
+	if val := r.Header.Get("Range"); val != "" {
+		// zero value count indicates from the offset to the resource's end, suffix-length is not required
+		input.Range = azblob.HTTPRange{Offset: 0, Count: 0}
```

Comment:
Does Azure support fetching multiple ranges in a single request? I.e. using `Range: <unit>=<range-start>-<range-end>, …, <range-startN>-<range-endN>` (from https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Range). S3 doesn't so the s3store doesn't support it. We can do the same here, but I was just wondering.

Reply:
> Regarding multiple range in a single request, that is not supported. see also https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/storage/azblob@v1.6.1/internal/exported#HTTPRange

Ok, I see. Can you please add a comment mentioning that Azure doesn't support multiple ranges and thus azurestore falls back to fetching the full resource, just like s3store does?